ietf/context_processors.py

```diff
@@ -51,3 +52,8 @@ def timezone_now(request):
     return {
         'timezone_now': timezone.now(),
     }
+
+def traceparent_id(request):
+    context_extras = {}
+    inject(context_extras)
+    return context_extras
```

Comment:
What would you think about...
```suggestion
    return {"opentelemetry": context_extras}
```
and then the template would refer to `opentelemetry.traceparent`? Might help avoid collisions and make clearer where the values came from when looking at the context.

Reply:
That works